### ISSUE TITLE
[Google Maps] Custom SVG marker misaligns when zooming out

### ISSUE THREAD

**Issue by Aadlei** [vikramlearning/blazorbootstrap#915]
I've had issues with custom svg markers (when using GoogleMapMarker.Content) where it misaligns when zoomed out (it goes away from its GoogleMapMarkerPosition, see pictures below:

Marker with accurate positioning (zoomed in):
![image](https://github.com/user-attachments/assets/6274c5c8-40cf-46d9-ad07-286a448ae368)

Marker that now misaligns from its position, where light green is where it was when zoomed in
![image](https://github.com/user-attachments/assets/47c53d3f-0912-4138-b10c-e37b36cb924f)

After some search, I found posts saying that [you have to define the anchor point of the SVG marker](https://stackoverflow.com/questions/24451305/google-maps-marker-alignment-issues-during-zooming-when-using-a-svg-marker).

This might also mean that my SVG marker might be too complicated (meant to be an 8-directional sensor measurer), but I still feel that this should be a feature a part of this repo, since there are probably more than me that uses custom HTML on these sort of stuff. Especially considering it works really well otherwise, let me know if I can assist further.

Thanks,
Aadlei.

**Comment by gvreddy04** [comment]
@Aadlei Thank you for bringing this to our attention. This feature would be a valuable addition. If possible, please consider creating a pull request.

**Comment by Aadlei** [comment]
> @Aadlei Thank you for bringing this to our attention. This feature would be a valuable addition. If possible, please consider creating a pull request.

I gave this a try, looked around and tried to apply a solution. It seems that the "anchor" was only specific to the v3 api, so it is no longer a factor.
And from that, I made a solution that is applicable to other people that is also using SVG's with weird structure as markers. 

`.custom-marker {
    position: absolute;
    transform: translate(-50%, -50%);
}`

Applying this to SVGs like so: `<svg class='custom-marker' width='100' height='100' viewBox='0 0 500 500'`
fixed my problem as seen below:
![image](https://github.com/user-attachments/assets/2feefdc5-78fa-4e1e-a17d-dc5973180ae9)

Thanks.
